sessions/sessions.go

```diff
@@ -0,0 +1,58 @@
+package sessions
+
+import (
+	"sync"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/percona/rds_exporter/config"
+)
+
+// Sessions is a pool of aws *session.Sessions.
+type Sessions struct {
+	sessions map[config.Instance]*session.Session
+	sync.RWMutex
```

Comment:
Do we need it to be embedded, exposing its methods? Can it be a normal field instead?

Reply:
I don't see a reason why this is an issue. It's pretty common practice if I recall, but maybe something changed. Feel free to do research which one is `Go style` right now.

https://talks.golang.org/2012/10things.slide#3

Reply:
This way Lock/Unlock methods becomes a part of a public type method set (interface) without a good reason. It is not clear if one should call `Lock` (or `RLock`?) before `Load` or not. 

The type in the link above is "dumb" – it does not have any other methods, it is just a container for accessing `n` with locks. This type is "smart" – it has other methods.